src/aap_eda/services/activation/manager.py

```diff
@@ -1059,54 +1054,51 @@ def _create_activation_instance(self):
             raise exceptions.ActivationStartError(msg) from exc
 
     def _build_container_request(self) -> ContainerRequest:
-        if self.db_instance.extra_var:
-            context = yaml.safe_load(self.db_instance.extra_var.extra_var)
-        else:
-            context = {}
+        try:
+            container_params = self.db_instance.get_container_parameters()
+        except ContainerableInvalidError:
+            msg = (
+                f"Activation {self.db_instance.id} not valid, "
+                "container request cannot be built."
+            )
+            LOGGER.exception(msg)
```

Comment:
@jshimkus-rh should this be LOGGER.error as opposed to LOGGER.exception otherwise it will raise an exception and kill the worker.

Reply:
I essentially copy/pasted the exception handling in `_build_cmdline` which was already using `LOGGER.exception`; I would think both should have the same behavior.  Being `error` is probably better for both, unless there's something special about the command line creation?